script/prepare-release.js

```diff
@@ -95,19 +97,44 @@ async function getReleaseNotes (currentBranch) {
       `${currentBranch}, skipping release.`)
     process.exit(0)
   }
+
+  let prCount = 0
   commitComparison.data.commits.forEach(commitEntry => {
     let commitMessage = commitEntry.commit.message
-    if (commitMessage.toLowerCase().indexOf('merge') > -1) {
-      let mergeRE = /Merge pull request (#\d+) from .*\n/
-      let prMatch = commitMessage.match(mergeRE)
-      commitMessage = commitMessage.replace(mergeRE, '').replace('\n', '')
-      if (commitMessage.substr(commitMessage.length - 1, commitMessage.length) !== '.') {
-        commitMessage += '.'
+    try {
+      if (commitMessage.toLowerCase().indexOf('#') > -1) {
+        let mergeRE = /Merge pull request #(\d+) from .*\n/
+        let prMatch = commitMessage.match(mergeRE)
+        let prNumber
+        if (prMatch) {
+          commitMessage = commitMessage.replace(mergeRE, '').replace('\n', '')
+          let newlineRE = /(.*)\n*.*/
+          let newlineMatch = commitMessage.match(newlineRE)
+          if (newlineMatch) {
+            commitMessage = newlineMatch[1]
+          }
+          prNumber = prMatch[1]
+        } else {
+          let prRE = /(.* )\(#(\d+)\)(?:.*)/
+          prMatch = commitMessage.match(prRE)
+          if (prMatch) {
+            commitMessage = prMatch[1].trim()
+            prNumber = prMatch[2]
+          }
+        }
+        if (prMatch) {
+          if (commitMessage.substr(commitMessage.length - 1, commitMessage.length) !== '.') {
+            commitMessage += '.'
+          }
+          releaseNotes += `* ${commitMessage} #${prNumber} \n\n`
+          prCount++
+        }
       }
-      releaseNotes += `${commitMessage} ${prMatch[1]} \n\n`
+    } catch (ex) {
+      console.log(`Ex: `, ex)
```

Comment:
If we ever catch this ex, we'll wish we had `commitMessage` in here too

Reply:
That try/catch shouldn't be there.  I was debugging something and forgot to take it out.